javascript/javascript1/week2/fullname.js

```diff
@@ -0,0 +1,15 @@
+function getFullname(firstName, surName, useFormalName, gender) {
+  if (useFormalName) {
+    if (gender === "male") {
+      return "Lord" + " " + firstName + " " + surName;
```

Comment:
Using string interpolation seems to have become the defacto standard in JavaScript... I recommend you to use it exclusively in the next homework assignment. (You don't need to change it here... I have seen you can do it in the candy assignment).

I could look something like:
```js
`Lord ${firstName} ${surName}`
```

Reply:
👍🏻 will do it